Sources/Purchasing/ProductsManager.swift

```diff
@@ -108,31 +108,11 @@ class ProductsManager: NSObject, ProductsManagerType {
         }
     }
 
-    func cache(_ product: StoreProductType) {
-        switch product {
-        case let sk1Product as SK1Product:
-            self.productsFetcherSK1.cacheProduct(sk1Product)
-
-        case let product as StoreProduct:
-            if let sk1Product = product.sk1Product {
-                self.productsFetcherSK1.cacheProduct(sk1Product)
-            }
-
-        default:
-            // Ignoring otherwise.
-            // This is temporary, as the caching logic is going away from here
-            // in favor of the new `CachingProductsManager`
-            break
-        }
-    }
-
+    // This class does not implement caching.
+    // See `CachingProductsManager`.
+    func cache(_ product: StoreProductType) {}
     func clearCache() {
-        if #available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *),
-           self.systemInfo.storeKit2Setting == .enabledForCompatibleDevices {
-            self.invalidateAndReFetchCachedSK2Products()
-        } else {
-            self.invalidateAndReFetchCachedSK1Products()
-        }
+        self.productsFetcherSK1.clearCache()
```

Comment:
I guess we want to remove the caching from the `ProductsFetcherSK1` in the future, since we are now caching it in the `CachingProductsManager`. Did we create a Jira task for it?

Reply:
Yeah that's why I wrote this:
> I've decided to keep the logic in ProductsFetcherSK1 because it's intertwined with the retry mechanism, and it wasn't trivial to remove caching while leaving that logic.

We could file a Jira to do it in the future, but I wonder if it's worth spending the time on that if SK1 is going away eventually /cc @aboedo thoughts?